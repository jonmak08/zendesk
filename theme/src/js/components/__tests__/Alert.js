import preact from 'preact';
import {cleanup, render} from 'preact-testing-library';

import Alert from '../Alert';

afterEach(cleanup);

describe('Alert', () => {
	it('renders correctly', () => {
		const {container} = render(<Alert>Alert</Alert>);

		expect(container).toMatchSnapshot();
	});

	it('renders with leading text', () => {
		const {container} = render(<Alert leadingText="Leading Text">Alert</Alert>);

		expect(container).toMatchSnapshot();
	});
});