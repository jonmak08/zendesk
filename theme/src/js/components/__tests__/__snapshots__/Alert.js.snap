// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Alert renders correctly 1`] = `
<div>
  <div
    class="alert alert-info"
    role="alert"
  >
    <span
      class="alert-indicator"
    >
      <svg
        class="lexicon-icon lexicon-icon-info-circle"
        role="img"
      >
        <use
          href="#info-circle"
        />
      </svg>
    </span>
    Alert
  </div>
</div>
`;

exports[`Alert renders with leading text 1`] = `
<div>
  <div
    class="alert alert-info"
    role="alert"
  >
    <span
      class="alert-indicator"
    >
      <svg
        class="lexicon-icon lexicon-icon-info-circle"
        role="img"
      >
        <use
          href="#info-circle"
        />
      </svg>
    </span>
    <strong
      class="lead"
    >
      Leading Text
    </strong>
    Alert
  </div>
</div>
`;
